java-frontend/src/main/java/org/sonar/java/se/MethodBehavior.java

```diff
@@ -83,6 +84,14 @@ private boolean isConstructor() {
     return ImmutableList.<MethodYield>builder().addAll(yields).build();
   }
 
+  Stream<MethodYield> exceptionalPathYields() {
+    return yields.stream().filter(y -> y.exception);
```

Comment:
wondering if you could make things clearer by declaring the predicate and use the negate method on it eventually : https://docs.oracle.com/javase/8/docs/api/java/util/function/Predicate.html#negate--